src/shared/database/entities/verificationCode.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { Column, Entity, ObjectID, ObjectIdColumn } from 'typeorm';
+import { Base } from './base.entity';
+
+@Entity()
+export class VerificationCodesEntity extends Base {
+  @ObjectIdColumn()
+  userId!: ObjectID;
```

Comment:
Try adding this and tell me if you can see any changes: `@Column(type => UserEntity)`